lib/dao.js

```diff
@@ -93,10 +93,11 @@ function applyStrictCheck(model, strict, data, inst, cb) {
     key = keys[i];
     if (props[key]) {
       result[key] = data[key];
-    } else if (strict === 'throw') {
-      cb(new Error(g.f('Unknown property: %s', key)));
-      return;
-    } else if (strict === 'validate') {
+    } else if (strict) {
+      if (strict === 'throw') {
+        g.warn('{{`strict: throw`}} has been removed, use {{`strict: true`}} instead ' +
```

Comment:
This will print a warning for each call of a DAO operation, which can flood the console output and also won't be triggered until an operation is invoked.

I am proposing to check `settings.strict` at the time the model is set up instead. That way the warning will be always printed only once for each model and immediately at startup time.

Also make sure to include model name in the error message, otherwise it's difficult to find which of the application models is having the issue.